src/test/app/PayChan_test.cpp

```diff
@@ -1160,6 +1160,50 @@ struct PayChan_test : public beast::unit_test::suite
             bob.human());
     }
 
+    void
+    testAccountChannelAuthorize(FeatureBitset features, unsigned int apiVersion)
+    {
+        testcase("PayChan Channel_Auth RPC Api " + std::to_string(apiVersion));
+        using namespace jtx;
+        using namespace std::literals::chrono_literals;
+
+        Env env{*this, features};
+        auto const alice = Account("alice");
+        auto const bob = Account("bob");
+        auto const charlie = Account("charlie", KeyType::ed25519);
+        env.fund(XRP(10000), alice, bob, charlie);
+        auto const pk = alice.pk();
+        auto const settleDelay = 3600s;
+        auto const channelFunds = XRP(1000);
+        auto const chan1Str = to_string(channel(alice, bob, env.seq(alice)));
+        env(create(alice, bob, channelFunds, settleDelay, pk));
+        env.close();
+
+        // test for api_version 2
+        Json::Value args{Json::objectValue};
+        args[jss::api_version] = apiVersion;
+        args[jss::channel_id] = chan1Str;
+        args[jss::key_type] = "ed255191";
+        args[jss::seed] = "snHq1rzQoN2qiUkC3XF5RyxBzUtN";
+        args[jss::amount] = 51110000;
+        if (apiVersion < 2u)
```

Comment:
Consider moving the testVersion loop inside this function. The channel above doesn't need to be created twice. Then this is going to be:
>     for (auto apiVersion = RPC::apiMinimumSupportedVersion;
>             apiVersion <= RPC::apiBetaVersion;
>             ++apiVersion)
>        {
>            testcase("PayChan Channel_Auth RPC Api " + std::to_string(apiVersion));
>            args[jss::api_version] = apiVersion;
>            auto const rs = env.rpc(
>                "json",
>                "channel_authorize",
>                args.toStyledString())[jss::result];
>            auto const error = apiVersion < 2u ? "invalidParams" : "badKeyType";
>            BEAST_EXPECT(rs[jss::error] == error);
>        }